<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="saveCurveButton">
      <property name="text">
       <string>Save curve</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="28" column="1">
       <widget class="QPushButton" name="readoutLossesButton">
        <property name="text">
         <string>Read-out losses</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLineEdit" name="filterCavityLengthBox">
        <property name="text">
         <string>85e-2</string>
        </property>
       </widget>
      </item>
      <item row="31" column="0">
       <widget class="QLineEdit" name="mmSqueezerLOBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QPushButton" name="ifoLengthButton">
        <property name="text">
         <string>Cavity length (m)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="injectionLossesButton">
        <property name="text">
         <string>Injection losses</string>
        </property>
       </widget>
      </item>
      <item row="31" column="1">
       <widget class="QPushButton" name="mmSqueezerLOButton">
        <property name="text">
         <string>Squeezer/ Local oscillator</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="injectionLossesBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="wavelengthButton">
        <property name="text">
         <string>Wavelength (nm)</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QPushButton" name="filterCavityLengthButton">
        <property name="text">
         <string>Cavity length (m)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item row="29" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mode mismatch</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLineEdit" name="filterCavityLossesBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QPushButton" name="ifoFinesseButton">
        <property name="text">
         <string>Finesse</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="squeezingAngleButton">
        <property name="text">
         <string>Squeezed quadrature (°)</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QPushButton" name="filterCavityLossesButton">
        <property name="text">
         <string>Cavity losses</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Squeezer</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLineEdit" name="ifoMassBox">
        <property name="text">
         <string>0.1e-9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="centerFreqButton">
        <property name="text">
         <string>Center frequency (Hz)</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLineEdit" name="ifoFinesseBox">
        <property name="text">
         <string>50000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="centerFreqBox">
        <property name="text">
         <string>1.5e6</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QPushButton" name="ifoMassButton">
        <property name="text">
         <string>Effective mass (kg)</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="inputTransmissionButton">
        <property name="text">
         <string>Input mirror transmission</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="inputIntensityBox">
        <property name="text">
         <string>5e15</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="inputTransmissionBox">
        <property name="text">
         <string>0.07</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="wavelengthBox">
        <property name="text">
         <string>1064</string>
        </property>
       </widget>
      </item>
      <item row="27" column="1">
       <widget class="QPushButton" name="homodyneAngleButton">
        <property name="text">
         <string>Homodyne angle (°)</string>
        </property>
       </widget>
      </item>
      <item row="27" column="0">
       <widget class="QLineEdit" name="homodyneAngleBox">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="spanFreqButton">
        <property name="text">
         <string>Frequency span (Hz)</string>
        </property>
       </widget>
      </item>
      <item row="24" column="0">
       <widget class="QLineEdit" name="ifoQualityFactorBox">
        <property name="text">
         <string>1e7</string>
        </property>
       </widget>
      </item>
      <item row="30" column="1">
       <widget class="QPushButton" name="mmSqueezerCavityButton">
        <property name="text">
         <string>Squeezer / Filter cavity</string>
        </property>
       </widget>
      </item>
      <item row="28" column="0">
       <widget class="QLineEdit" name="readoutLossesBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Laser</string>
        </property>
       </widget>
      </item>
      <item row="30" column="0">
       <widget class="QLineEdit" name="mmSqueezerCavityBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Interferometer</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="squeezingFactorBox">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="23" column="0">
       <widget class="QLineEdit" name="ifoOmegamBox">
        <property name="text">
         <string>1.5e6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="spanFreqBox">
        <property name="text">
         <string>1.5e6</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="inputIntensityButton">
        <property name="text">
         <string>Input intensity (photons/s)</string>
        </property>
       </widget>
      </item>
      <item row="23" column="1">
       <widget class="QPushButton" name="ifoOmegamButton">
        <property name="text">
         <string>Mech. resonance frequency (Hz)</string>
        </property>
       </widget>
      </item>
      <item row="26" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Read-out</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="detuningButton">
        <property name="text">
         <string>Detuning (Hz)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter cavity</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLineEdit" name="detuningBox">
        <property name="text">
         <string>1.5e6</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <widget class="QPushButton" name="propagationLossesButton">
        <property name="text">
         <string>Propagation losses</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="squeezingAngleBox">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
      <item row="25" column="0">
       <widget class="QLineEdit" name="propagationLossesBox">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLineEdit" name="ifoLengthBox">
        <property name="text">
         <string>100e-6</string>
        </property>
       </widget>
      </item>
      <item row="24" column="1">
       <widget class="QPushButton" name="ifoQualityFactorButton">
        <property name="text">
         <string>Mechanical quality factor</string>
        </property>
       </widget>
      </item>
      <item row="32" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="squeezingFactorButton">
        <property name="text">
         <string>Squeezing factor (dB)</string>
        </property>
       </widget>
      </item>
      <item row="33" column="0" colspan="2">
       <widget class="QCheckBox" name="unitBox">
        <property name="text">
         <string>Units in m^2/Hz (if unchecked, prints dB under shot noise)</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLineEdit" name="filterCavityFinesseBox"/>
      </item>
      <item row="16" column="0">
       <widget class="QLineEdit" name="filterCavityBWBox"/>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="filterCavityFinesseButton">
        <property name="text">
         <string>Finesse</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QPushButton" name="filterCavityBWButton">
        <property name="text">
         <string>Bandwidth (Hz)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="saveParametersButton">
      <property name="text">
       <string>Save parameters</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="loadParametersButton">
      <property name="text">
       <string>Load parameters</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="4">
     <widget class="PlotWidget" name="graph"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
